omniscidb/DataMgr/Encoder.cpp

```diff
@@ -60,7 +60,7 @@ Encoder* Encoder::Create(Data_Namespace::AbstractBuffer* buffer,
         case 8:
           return new NoneEncoder<int64_t>(buffer);
         default:
-          CHECK(false);
+          CHECK("Unhandled primitive type size" && false);
```

Comment:
I don't think we need any messages here. But anyway, it is done through streaming for our checks: `CHECK(false) << "Error message."`. And if still want to have a message here, then better to print the type (`toString` gives the full info about the type) causing the problem.